src/Web/Avalonia.Blazor/wwwroot/SKHtmlCanvas.js.map

```diff
@@ -0,0 +1 @@
AER,UAAU,CAAC,YAAY,CAAC,qBAAqB,CAAC,UAAU,EAAE,KAAK,EAAE,MAAM,CAAC,CAAC;IAC1E,CAAC;IAEM,MAAM,CAAC,mBAAmB,CAAC,OAA0B,EAAE,MAAe;QAC5E,MAAM,UAAU,GAAG,OAA8B,CAAC;QAClD,IAAI,CAAC,UAAU,IAAI,CAAC,UAAU,CAAC,YAAY;YAC1C,OAAO;QAER,UAAU,CAAC,YAAY,CAAC,mBAAmB,CAAC,MAAM,CAAC,CAAC;IACrD,CAAC;IAEM,MAAM,CAAC,YAAY,CAAC,OAA0B,EAAE,KAAa,EAAE,KAAa,EAAE,MAAc;QAClG,MAAM,UAAU,GAAG,OAA8B,CAAC;QAClD,IAAI,CAAC,UAAU,IAAI,CAAC,UAAU,CAAC,YAAY;YAC1C,OAAO;QAER,UAAU,CAAC,YAAY,CAAC,YAAY,CAAC,KAAK,EAAE,KAAK,EAAE,MAAM,CAAC,CAAC;IAC5D,CAAC;IA4BM,MAAM;QACZ,IAAI,CAAC,mBAAmB,CAAC,KAAK,CAAC,CAAC;IACjC,CAAC;IAEM,qBAAqB,CAAC,UAAoB,EAAE,KAAc,EAAE,MAAe;QACjF,oCAAoC;QACpC,IAAI,UAAU,KAAK,SAAS,IAAI,IAAI,CAAC,iBAAiB,KAAK,UAAU;YACpE,IAAI,CAAC,mBAAmB,CAAC,UAAU,CAAC,CAAC;QAEtC,2DAA2D;QAC3D,IAAI,KAAK,IAAI,MAAM,EAAE;YACpB,IAAI,CAAC,UAAU,CAAC,KAAK,GAAG,KAAK,CAAC;YAC9B,IAAI,CAAC,UAAU,CAAC,MAAM,GAAG,MAAM,CAAC;SAChC;QAED,qCAAqC;QACrC,IAAI,IAAI,CAAC,iBAAiB,KAAK,CAAC;YAC/B,OAAO;QAER,iCAAiC;QACjC,IAAI,CAAC,iBAAiB,GAAG,MAAM,CAAC,qBAAqB,CAAC,GAAG,EAAE;YAC1D,IAAI,IAAI,CAAC,MAAM,EAAE;gBAChB,eAAe;gBACf,EAAE,CAAC,kBAAkB,CAAC,IAAI,CAAC,MAAM,CAAC,OAAO,CAAC,CAAC;aAC3C;YAED,IAAI,CAAC,mBAAmB,CAAC,YAAY,CAAC,QAAQ,CAAC,CAAC;YAChD,IAAI,CAAC,iBAAiB,GAAG,CAAC,CAAC;YAE3B,qCAAqC;YACrC,IAAI,IAAI,CAAC,iBAAiB;gBACzB,IAAI,CAAC,qBAAqB,EAAE,CAAC;QAC/B,CAAC,CAAC,CAAC;IACJ,CAAC;IAEM,mBAAmB,CAAC,MAAe;QACzC,IAAI,CAAC,iBAAiB,GAAG,MAAM,CAAC;QAEhC,wDAAwD;QACxD,IAAI,MAAM,EAAE;YACX,wFAAwF;YACxF,IAAI,CAAC,qBAAqB,EAAE,CAAC;SAC7B;aAAM,IAAI,IAAI,CAAC,iBAAiB,KAAK,CAAC,EAAE;YACxC,MAAM,CAAC,oBAAoB,CAAC,IAAI,CAAC,iBAAiB,CAAC,CAAC;YACpD,IAAI,CAAC,iBAAiB,GAAG,CAAC,CAAC;SAC3B;IACF,CAAC;IAEM,YAAY,CAAC,KAAa,EAAE,KAAa,EAAE,MAAc;QAC/D,IAAI,IAAI,CAAC,MAAM,IAAI,CAAC,KAAK,IAAI,KAAK,IAAI,CAAC,IAAI,KAAK,IAAI,CAAC;YACpD,OAAO,KAAK,CAAC;QAEd,IAAI,GAAG,GAAG,IAAI,CAAC,UAAU,CAAC,UAAU,CAAC,IAAI,CAAC,CAAC;QAC3C,IAAI,CAAC,GAAG,EAAE;YACT,OAAO,CAAC,KAAK,CAAC,qCAAqC,CAAC,CAAC;YACrD,OAAO,KAAK,CAAC;SACb;QAED,2DAA2D;QAC3D,IAAI,CAAC,UAAU,CAAC,KAAK,GAAG,KAAK,CAAC;QAC9B,IAAI,CAAC,UAAU,CAAC,MAAM,GAAG,MAAM,CAAC;QAEhC,iCAAiC;QACjC,IAAI,MAAM,GAAG,IAAI,iBAAiB,CAAC,MAAM,CAAC,MAAM,CAAC,MAAM,EAAE,KAAK,EAAE,KAAK,GAAG,MAAM,GAAG,CAAC,CAAC,CAAC;QACpF,IAAI,SAAS,GAAG,IAAI,SAAS,CAAC,MAAM,EAAE,KAAK,EAAE,MAAM,CAAC,CAAC;QACrD,GAAG,CAAC,YAAY,CAAC,SAAS,EAAE,CAAC,EAAE,CAAC,CAAC,CAAC;QAElC,OAAO,IAAI,CAAC;IACb,CAAC;IAED,MAAM,CAAC,kBAAkB,CAAC,UAA6B;QACtD,MAAM,iBAAiB,GAAG;YACzB,KAAK,EAAE,CAAC;YACR,KAAK,EAAE,CAAC;YACR,OAAO,EAAE,CAAC;YACV,SAAS,EAAE,CAAC;YACZ,kBAAkB,EAAE,CAAC;YACrB,qBAAqB,EAAE,CAAC;YACxB,+BAA+B,EAAE,CAAC;YAClC,4BAA4B,EAAE,CAAC;YAC/B,YAAY,EAAE,CAAC;YACf,YAAY,EAAE,CAAC;YACf,yBAAyB,EAAE,CAAC;YAC5B,mBAAmB,EAAE,CAAC;YACtB,4BAA4B,EAAE,CAAC;SAC/B,CAAC;QAEF,IAAI,GAAG,GAA0B,EAAE,CAAC,aAAa,CAAC,UAAU,EAAE,iBAAiB,CAAC,CAAC;QACjF,IAAI,CAAC,GAAG,IAAI,iBAAiB,CAAC,YAAY,GAAG,CAAC,EAAE;YAC/C,OAAO,CAAC,IAAI,CAAC,2BAA2B,CAAC,CAAC;YAC1C,iBAAiB,CAAC,YAAY,GAAG,CAAC,CAAC;YACnC,iBAAiB,CAAC,YAAY,GAAG,CAAC,CAAC;YACnC,GAAG,GAAG,EAAE,CAAC,aAAa,CAAC,UAAU,EAAE,iBAAiB,CAAC,CAAC;SACtD;QAED,OAAO,GAAG,CAAC;IACZ,CAAC;CACD"}
```

Comment:
That's a generated file, consider adding `*.map` to `.gitignore`

Reply:
done.